win32/src/winapi/gdi32/bitmap.rs

```diff
@@ -152,6 +152,11 @@ pub fn BitBlt(
     y1: i32,
     rop: u32,
 ) -> bool {
+    if rop == BLACKNESS {
+        // It seems like passing null as `hdcSrc` when using BLACKNESS is supported on Windows.
```

Comment:
Looking at the docs, it's plausible to me that maybe all of the modes that don't involve one of the SRC* variants don't care about hdcsrc?

(You don't need to change this, just thinking out loud)

Reply:
...also, sorry for my terrible clunky bitmap code, I still don't have the right mental model about how it ought to work for me to write it better :(

Reply:
Yeah, it was a bit hard to follow the code, but I don't really know how to do it better 😅 

First I started wrapping the source in an `Option` but it became very messy, not sure how to best do this 🤔